src/observer-locator.js

```diff
@@ -81,7 +89,8 @@ export class ObserverLocator {
     return value;
   }
 
-  addAdapter(adapter: ObjectObservationAdapter) {
+  /**@param {ObjectObservationAdapter} adapter */
+  addAdapter(adapter) {
```

Comment:
Is there some reason that you remove the typing?

Reply:
probably to make the file javascript only, while preserving typing via jsDoc as there are no type annotation in the rest of the file. If can be either way for TypeScript, which is what I thought we use for generating typings. But since we are crafting to definition manually, this should be ok.

Reply:
If this PR is good to merge then I can push another commit to revert the typing here, also to adjust the typing in `CheckedObserver`, `SelectObserver` introduced in #656

Reply:
@EisenbergEffect I already fixed the typing issue in those observer constructors, so dis regard my previous comment.